frontend/components/ActivityMap.vue

```diff
@@ -0,0 +1,248 @@
+        ? "mapbox://styles/mathematti/clw4z6v0s028p01o0askbhfh9"
+        : "mapbox://styles/mathematti/clw4znxzh02mh01qz5hgk3qb6",
+);
+</script>
+
+<template>
+    <div class="flex items-center justify-center md:justify-start">
+        <div
+            class="relative mt-5 flex w-[90%] items-center sm:w-5/6 md:ml-[10%] md:w-[calc(50%+16rem)] md:justify-between lg:ml-10 lg:w-[calc(33.33vw+38.5rem)] xl:ml-[10%] xl:w-[calc(33.33vw+44rem)]"
+        >
+            <div class="-mb-2.5 text-2xl font-semibold">
+                <T key-name="journey.map" />
+            </div>
+            <span
+                v-if="activitiesWithoutLocation.length > 0"
+                v-tooltip.top="{
+                    value: t('journey.map.notfound.tooltip'),
+                    pt: { root: 'font-nunito' },
+                }"
+                class="pi pi-exclamation-circle -mb-2.5 ml-auto text-xl text-cancel-border hover:cursor-pointer"
+                @click="isNotFoundActivitiesDialogVisible = true"
+            />
+        </div>
+    </div>
+    <div class="flex justify-center md:justify-start">
+        <div
+            class="relative mt-5 flex h-96 w-[90%] items-end sm:w-5/6 md:ml-[10%] md:w-[calc(50%+16rem)] md:justify-start lg:ml-10 lg:w-[calc(33.33vw+38.5rem)] xl:ml-[10%] xl:w-[calc(33.33vw+44rem)]"
+        >
+            <MapboxMap
+                :map-id="journey.getID()"
+                style="position: absolute; top: 0; bottom: 0"
+                class="rounded-xl"
+                :options="{
+                    style: style,
+                    center: [long, lat],
+                    zoom: zoom,
+                }"
+            >
+                <MapboxDefaultMarker
+                    v-for="activity in activitiesWithLocation"
+                    :key="activity.id"
+                    :marker-id="activity.id"
+                    :lnglat="[activity.longitude, activity.latitude]"
+                >
+                    <MapboxDefaultPopup
+                        :popup-id="activity.id"
+                        :lnglat="[activity.longitude, activity.latitude]"
+                        :options="{
+                            closeOnClick: true,
+                        }"
+                    >
+                        <div
+                            class="flex flex-col font-nunito text-text dark:text-input"
+                        >
+                            <h1 class="font-bold">
+                                {{ activity.name }}
+                            </h1>
+                            <p>{{ activity.mapbox_full_address }}</p>
+                        </div>
+                    </MapboxDefaultPopup>
+                </MapboxDefaultMarker>
+
+                <MapboxScaleControl />
+                <MapboxGeolocateControl />
+                <MapboxFullscreenControl />
+                <MapboxNavigationControl />
+            </MapboxMap>
+
+            <Dialog
```

Comment:
![image](https://github.com/JourneyPlanner/JourneyPlanner/assets/125883002/8162d8c5-9b11-45f2-a44a-da11ade33e43)
Für das Pop up (beim Laptop, Handy passt)
-> Die Schrift größe bei der Tabelle zumindest eine Schriftgröße größer
-> die linke seite der Tabelle näher an die rechte (nicht in der mitte teilen -> zu viel white space)
-> größe des Pop ups so dass 5 inputs OHNE zu scrollen sichtbar sind
-> Abstand ist zwischen den Infos (Aktivitäten Name, angegebene Adresse) zu groß -> kleiner machen
-> "Wir konnten diese Aktivität ....." -Text weiter nach oben rücken (auch für die Handy Version bitte)